mlf_core/create/templates/mlflow/mlflow_pytorch/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug_no_hyphen }}/data_loading/data_loader.py

```diff
@@ -1,21 +1,49 @@
-import torch
-
+import pytorch_lightning as pl
 from torchvision import datasets, transforms
+from torch.utils.data import DataLoader
+
+
+class MNISTDataModule(pl.LightningDataModule):
+    def __init__(self, **kwargs):
+        """
+        Initialization of inherited lightning data module
```

Comment:
That's a pretty useless comment

Reply:
Was already there, will remove

Reply:
You can also make it more specific than removing it :)